public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -316,7 +316,9 @@ block add-new-hero-via-detail-comp
   The user can *delete* an existing hero by clicking a delete button next to the hero's name.
   Add the following to the heroes component HTML right after the hero name in the repeated `<li>` tag:
 +makeExample('app/heroes.component.html', 'delete')
-
+:marked
+  Add the following CSS to the bottom of `heroes.component.css`.
```

Comment:
Alrighty

On Tue, Aug 2, 2016, 12:10 PM Patrice Chalin <notifications@github.com>
wrote:

> In public/docs/ts/latest/tutorial/toh-pt6.jade
> <https://github.com/angular/angular.io/pull/1751#discussion_r73196526>:
>
> > @@ -316,7 +316,9 @@ block add-new-hero-via-detail-comp
> >    The user can *delete* an existing hero by clicking a delete button next to the hero's name.
> >    Add the following to the heroes component HTML right after the hero name in the repeated `<li>` tag:
> >  +makeExample('app/heroes.component.html', 'delete')
> > -
> > +:marked
> > +  Add the following CSS to the bottom of `heroes.component.css`.
>
> Oh, I just noticed this. We need to make this sentence work for all
> language versions. I'd suggest going with:
>
> Add the following to the bottom of the HeroesComponent CSS file:
> +makeExcerpt('app/heroes.component.css', 'additions')
>
> (Drop the last '' argument to makeExcerpt.)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/angular/angular.io/pull/1751/files/0320013bbf27da52ff1e3bd7bd7b7d155b697be2#r73196526>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA-2UIHQBheeyaJ9PhZkxsF3UJxBto5Eks5qb3nqgaJpZM4I-i5r>
> .
>